documentation/docs/guide/GettingStarted/building_images.md

```diff
@@ -1,46 +1,92 @@
-# Building Images
-Base images are images that would use dockerhub image streams to create an additional layer to add more functionality or modify existing functionality of the image.
+# Container Images
 
-**IMPORTANT** There are multi-stage build, which these are only part of the build.  This documentation is incomplete, but left here as a starting point.
+## Base Images on OpenShift
+::: tip
+Updating base images on OpenShift is only done at the specific Build config;
+-  [s2i Node](https://github.com/bcgov/gdx-agreements-tracker/blob/development/deployments/kustomize/image-builds/nodejs-s2i.yaml#L29)
```

Comment:
![image](https://github.com/bcgov/gdx-agreements-tracker/assets/122112933/74c787af-c67c-4b81-a16d-2ad514b178f6)
the punctuation is a bit off and the triple-colons are rendered literally

Reply:
pretty sure this will resolve itself with vuepress,